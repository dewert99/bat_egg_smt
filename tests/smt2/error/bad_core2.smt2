(assert true)
(check-sat)
(get-unsat-core)