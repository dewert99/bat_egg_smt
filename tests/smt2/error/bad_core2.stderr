The last command was not `check-sat-assuming` that returned `Unsat` at 3:2
(get-unsat-core)
 ^

